core/primitives/src/upgrade_schedule.rs

```diff
@@ -171,12 +192,18 @@ impl ProtocolUpgradeVotingSchedule {
     // The format is comma separate datetime:=version pairs.
     fn parse_override(
         override_str: &str,
+        min_supported_protocol_version: ProtocolVersion,
+        client_protocol_version: ProtocolVersion,
     ) -> Result<ProtocolUpgradeVotingScheduleRaw, ProtocolUpgradeVotingScheduleError> {
         // The special value "now" means that the upgrade should happen immediately.
         if override_str.to_lowercase() == "now" {
             return Ok(vec![]);
         }
 
+        if override_str.to_lowercase() == "one_by_one" {
+            return Ok(Self::one_by_one(min_supported_protocol_version, client_protocol_version));
+        }
```

Comment:
this looks more idiomatic to me.
```suggestion
        match override_str.to_lowercase() {
            "now" => return Ok(vec![]),
            "one_by_one" => return Ok(Self::one_by_one(
                min_supported_protocol_version, 
                client_protocol_version)),
            _ => {},
        }
```